samples/05_content_publishers/pdf_table_to_pdf_map.ipynb

```diff
@@ -0,0 +1,1459 @@
+{
```

Comment:
<p>Could you check if CAD stands for "Computer Aided Dispatch (CAD) system"? I thought it stood for Computer Aided Drawing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/541/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>